models/vrp.rb

```diff
@@ -124,7 +126,7 @@ def self.create(hash, delete = true)
       vrp = super({})
       self.filter(hash) # TODO : add filters.rb here
       # moved filter here to make sure we do have schedule_indices (not date) to do work_day check with lapses
-      self.check_consistency(hash)
+      vrp.check_consistency(hash)
```

Comment:
See conversation https://github.com/Mapotempo/optimizer-api/pull/181#discussion_r617300541
(I am sorry, I force pushed the branch when previous PR was closed and I could not open PR181 again!)

Reply:
@senhalil @braktar did we agree on keeping this as is because it is meant to be changed anyway ?